pkg/policy/repository.go

```diff
@@ -288,14 +302,10 @@ func (p *Repository) ResolveL4IngressPolicy(ctx *SearchContext) (*L4PolicyMap, e
 		}
 	}
 
-	if result != nil {
```

Comment:
Maybe this should be kept here as we returned `L4PolicyMap` should contain the revision number when the L4PolicyMap was created, this should not be taken care by the caller of this function.

Reply:
Done.